src/service/packet_verifier.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+
+package helium.packet_verifier;
+
+message valid_packet {
+  uint32 payload_size = 1;
+  bytes gateway = 2;
+  bytes payload_hash = 3;
+}
+
+message invalid_packet {
```

Comment:
no invalid reason to include? A decision was made to have it be invalid so we should declare why to avoid losing the context.

Reply:
There will be gaming detection so I expect a number of reasons to crop up over time. Follow the model the iot verifier uses with a proto error enum ?

Reply:
Done